content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md

```diff
@@ -0,0 +1,131 @@
+---
+title:  Organizando o acesso ao cluster usando arquivos kubeconfig
+content_type: conceito
```

Comment:
We don't need to translate the content-type. The correct is *concept*.

Reply:
thanks for the info. already updated on 9f795b4